front-end/src/components/PostListItem.tsx

```diff
@@ -2,6 +2,7 @@ import React, { useState } from 'react';
 import { Button, Card, CardBody, CardLink, CardSubtitle, CardText, CardTitle } from 'reactstrap';
 import { Post } from '../types/Post';
 import { UserLogin } from '../types/UserLogin';
+import ContentPreview from './PostContentEl';
```

Comment:
nit. In different files the naming is different, like in PostDetailModal its called PostContent, maybe just change the file and component name to just be ContentPreview or something and keep it consistent in the imports through all the files. I'm also not sure what the El at the end stands for.

Reply:
Good catch. I was renaming the component a lot, and that slipped through the cracks. El at the end stands for `Element` to avoid it being confused with the `PostContent` type. I'll see if I can have `PostContent` refer to both a type and component

Reply:
Awesome sounds good. Great stuff on the preview. Looks good 👍

Reply:
After trying it out, looks like there's a naming conflict if both the component and type is named `PostContent`. I'll keep it as `PostContentEl` for now, until we find a better name. IMO `El` is an unusual suffix